### ISSUE TITLE
Wrong indention level of EQ_SUB token

### ISSUE THREAD

**Issue by lorenzwalthert** [r-lib/styler#352]
5 and b should be indented 2 spaces.
``` r
styler::style_text("call(a = \n 5, \nb)", scope = "indention")
#> call(a =
#>     5,
#>   b)
```

Created on 2018-02-21 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

**Comment by lorenzwalthert** [comment]
Blocks #351

**Comment by lorenzwalthert** [comment]
Solved in #353 but this is blocked by #352.

**Comment by lorenzwalthert** [comment]
Solved in #353 but this is blocked by #351.

**Comment by lorenzwalthert** [comment]
Closed with #353.
